### PR TITLE
Add Prev/Next page actions to GestureManager

### PR DESCRIPTION
**Pull request by NiLuJe**
c.f., https://www.mobileread.com/forums/showthread.php?t=315023

### CODE REVIEW

**Comment by NiLuJe** [comment]
Also changed the internal action name, because PgUp generally goes *backwards* on a keyboard ;).

File: frontend/apps/reader/modules/readergesture.lua [hunk 1]
```diff
@@ -191,10 +193,14 @@ function ReaderGesture:gestureAction(action)
         UIManager:setDirty("all", "full")
     elseif action == "bookmarks" then
         self.ui:handleEvent(Event:new("ShowBookmark"))
-    elseif action == "page_update_up10" then
+    elseif action == "page_jmp_fwd_10" then
         self:pageUpdate(10)
-    elseif action == "page_update_down10" then
+    elseif action == "page_jmp_fwd_1" then
+        self:pageUpdate(1)
```
Comment (Frenzie) [inline frontend/apps/reader/modules/readergesture.lua]: On EPUB this works as expected (at least in page-separated mode) but on PDF/DjVu this specific command definitely isn't the right one. It's supposed to move the display by a screen length, not to literally go to the next page in the document.

Comment (Frenzie) [reply frontend/apps/reader/modules/readergesture.lua]: https://github.com/koreader/koreader/blob/8189945be93b9951a18f6de9a3b7b9b0cd97fb7a/frontend/apps/reader/modules/readerrolling.lua#L393-L401

Comment (NiLuJe) [reply frontend/apps/reader/modules/readergesture.lua]: Stupid idea: send a "TapForward" event instead?

Comment (Frenzie) [reply frontend/apps/reader/modules/readergesture.lua]: Doesn't sound that stupid. ReaderPaging also does one that' called "Rel", just a different one.

```
function ReaderPaging:onTapForward()
    self:onPagingRel(1)
    return true
end

function ReaderPaging:onTapBackward()
    self:onPagingRel(-1)
    return true
end
```

**Pull request merged**